### DISCUSSION TITLE
miactl RESOURCE-TYPE

### DISCUSSION THREAD

**Discussion by giulioroggero** [mia-platform/community#343] [Q&A] [answered]
### Which section?

Console

### Which is the version?

11.7.1

### Description

using `miactl` to list inspect runtime I've some difficulties about the following commands

`miactl runtime list --environment DEV`
he ask me for  **RESOURCE-TYPE**
` miactl runtime list RESOURCE-TYPE [flags]`
where I can find the complete list of  RESOURCE-TYPE? I suppose that is the same of k8s resource type?

the second command is 
`miactl runtime logs --environment DEV`
he ask me for **resource-query** 
`miactl runtime logs resource-query [flags]`
is the application name that is listed by miactl runtime list? It seems works.

I suggest to improve the CLI help and the documentation.

Thanks!

**Comment by JGiola** [comment, marked as answer]
Your reasoning for the log command is correct but it is valid only for the `pods` resource; the `list` command currently support only the resources that you can see via the Console UI in the runtime section.

We have improved the documentation and helpers for the two commands. We will also add a new command that will dynamically show what are the currently supported resource types that you can use with the `runtime list` command so a CLI user can list them without using the UI.

They will be available with the next version of the CLI.

**Reply by giulioroggero** [reply]
Thanks @JGiola !
